modules/scheduler/batch-job-template/main.tf

```diff
@@ -94,3 +94,17 @@ resource "local_file" "job_template" {
   content  = local.job_template_contents
   filename = local.job_template_output_path
 }
+
+resource "null_resource" "submit_job" {
+  depends_on = [ local_file.job_template ]
```

Comment:
Because the "gcloud batch jobs submit" command doesn't include an explicit job ID (this is possible with gcloud but I didn't include it in the command in main.tf) Batch will generate a new unique job ID for each submission, and the multiple jobs will not conflict.

Reply:
I think we could improve usability a little by picking a name that is something like <job_id>-<random_hash>. This could still allow for multiple submits by picking a new hash each time but would be a little more clear where the job came from.

Right now it just shows up as `job-<long uid>`

All that said. I would do that in a separate PR and submit this as is.